### ISSUE TITLE
Questions about performance , loss and data augmentation

### ISSUE THREAD

**Issue by sutzu** [Adamdad/keras-YOLOv3-mobilenet#28]
Hi,
Thanks for the work.

I'm using train_Mobilenet.py to train train.txt of VOC2007, and try to reproduce the performance on your page (64.22%).
A question first is that, for VOC 2007, do you use the same setting in train_Mobilenet.py (input size, #epoch, etc.) to get the 64.22% model?
What I did is only changing the input size to 416 to fit anchor size and train on the training data of VOC 2007 (no validation data) and validate on 300 images of validation data.
The training loss I get is around 15 and validation loss is around 20 and cannot get lower.
mAP is around 52%.

I have tried increasing the epoch but the result only improve a little.
Did I miss anything? Or there are some other tricks need to be applied to make the result better.
I've been stuck at this for a while, and I'll be very grateful if you can help me with it.

I also take a look of the data generation and found that the data is randomly change size but not keeping the original size ones.
Is there any reason of not keeping them?

Thank you very much for the help.
